.run/CoreApiApplication.run.xml

```diff
@@ -0,0 +1,18 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
I guess we forgot to ignore this file, it seems like this is your custom-run profile